lib/facade/IMonitoringAspect.ts

```diff
@@ -63,7 +64,9 @@ export interface MonitoringAspectProps {
   readonly kinesisFirehose?: MonitoringAspectType<KinesisFirehoseMonitoringOptions>;
   readonly lambda?: MonitoringAspectType<LambdaFunctionMonitoringOptions>;
   readonly openSearch?: MonitoringAspectType<OpenSearchClusterMonitoringOptions>;
-  readonly rds?: MonitoringAspectType<RdsClusterMonitoringOptions>;
+  readonly rds?: MonitoringAspectType<
+    RdsClusterMonitoringOptions | RdsInstanceMonitoringOptions
```

Comment:
This would probably make it hard to specify options for both if you wanted to. How about a separate one for `rdsInstance`?

We could also add a `rdsCluster` that takes precedence over `rds` and mark `rds` as deprecated if the naming seems confusing.